first draft new refund info (#1130)
MultiSafepay/docs: content/integrations/hosted/lightspeed_app/faq/refunding-lightspeed.md

meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
read_more: "."
---

> Yes you can refund orders.  By default you can create a refund in your [MultiSafepay Control](https://merchant.multisafepay.com), however, refunding directly from your Lightspeed backend is possible when this setting is enabled.

Reviewer: ```suggestion
Yes, you can refund orders.  By default, you can create a refund in your [MultiSafepay Control](https://merchant.multisafepay.com). To enable refunding from your Lightspeed backend, follow the steps below.

```

{{< screen src="../img/status-refund-dots.png" class="small-img desktop-radius" caption="Example of a succesfull and awaiting  credit invoice refund">}}

We also put a message in the notes section of the transaction to inform you about the refund. If a refund cannot be processed or an error occured this will be shown in the notes section. 
No worries, the customer can't see this field. We will not try to redo a refund if it fails. 
If you want to refund a previously failed refund, please try again via the MultiSafepay Merchant Panel.

{{< screen src="../img/status-refund-notes.png" class="small-img desktop-radius" caption="Example note succesfull refund">}}

_Please note it will take longer till the customer sees money returned to their back account._

### How long does it take before a customer get their money back?
The time of processing varies per payment method. For more info please visit our [page about refunds](https://docs.multisafepay.com/faq/refunds/how-long-does-a-refund-take-to-process/).

### I Created a refund in the MultiSafepay Merchant panel, can I still refund via Lightspeed?

> Yes. MultiSafepay refunds cannot exceed the maximal transaction value. If a partial refund is done in the Merchant panel and later a refund is done via Lightspeed we will calculate maximum amount possible and use this amount if the original amount exceeds the maximal transaction value.

Author: xD
